### ISSUE TITLE
Feature: add recursive option for folder

### ISSUE THREAD

**Issue by Shooter3k** [Cynosureprime/rling#26]
when running a command like this, would it be possible to add a recursive option so that it can check files within subfolders? 

something like: 
rling -r big-file.txt new-file.txt /path/to/old-file.txt /path/to/others/* 

would check:
/path/to/others/* 
/path/to/others/test1/* 
/path/to/others/test2/* 
/path/to/others/test3/*

**Comment by Waffle2** [comment]
Yes.  You would need to either modify the post-option-processing argument list to expand it, or deal with each directory as it appears, around:

2860         if (sb1.st_mode & S_IFDIR) {
2861             fprintf(stderr,"skipping directory\n");
2862             continue;
2863         }

Personally, I think that doing the post-option-processing scan is the best way:  in that case, do it while it is checking for missing files, around:

2315         for (x=2; x<argc; x++) {
2316             if (strcmp(argv[x],"stdin") == 0 || strcmp(argv[x],"stdout") ==      0)
2317                 continue;
2318             if (stat(argv[x],&sb1)) {
2319                 fprintf(stderr,"File \"%s\" not found.  Aborting (see -i option)\n",argv[x]);
2320                 exit(1);
2321             }
2322         }
2323     }
